common/src/main/java/org/apache/uniffle/common/metrics/GRPCMetrics.java

```diff
@@ -38,11 +45,36 @@ public abstract class GRPCMetrics {
   public void register(CollectorRegistry collectorRegistry) {
     if (!isRegister) {
       metricsManager = new MetricsManager(collectorRegistry);
+      registerGeneralMetrics();
       registerMetrics();
       isRegister = true;
     }
   }
 
+  private void registerGeneralMetrics() {
```

Comment:
Could we put these metrics to the method `registerMetrics`?

Reply:
These metrics are sharable for ShuffleServerGrpcMetrics and CoordinatorGrpcMetrics. I think there is no need to init twice time.

Reply:
OK.